modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -1550,6 +1550,86 @@ public void testPhraseMatch() throws Exception {
 				" this looks like a kind of coca drink]");
 	}
 
+	@Test
+	public void testSchedulingAware() throws Exception {
+		_updateConfigurationJSON(
+			"queryConfiguration", JSONUtil.put("applyIndexerClauses", false));
+
+		Calendar calendar = CalendarFactoryUtil.getCalendar();
+
+		calendar.add(Calendar.DAY_OF_MONTH, +1);
+
+		Date displayDate = calendar.getTime();
+
+		_journalArticles.add(
+			JournalTestUtil.addArticle(
+				_group.getGroupId(), 0,
+				PortalUtil.getClassNameId(JournalArticle.class),
+				StringPool.BLANK, true,
+				HashMapBuilder.put(
+					LocaleUtil.US, "Article Scheduled"
+				).build(),
+				null,
+				HashMapBuilder.put(
+					LocaleUtil.US, StringPool.BLANK
+				).build(),
+				null, LocaleUtil.getSiteDefault(), displayDate, null, false,
+				true, _serviceContext));
+
+		_updateElementInstancesJSON(
+			new Object[] {
+				HashMapBuilder.<String, Object>put(
+					"boost", 1
+				).put(
+					"fields", new String[] {"title_${context.language_id}^2"}
+				).put(
+					"fuzziness", "AUTO"
+				).put(
+					"keywords", "${keywords}"
+				).put(
+					"minimum_should_match", 0
+				).put(
+					"operator", "or"
+				).put(
+					"slop", 0
+				).put(
+					"type", "best_fields"
+				).build()
+			},
+			new String[] {"Text Match Over Multiple Fields"});
+
+		_keywords = "Article";
+
+		_assertSearch("[Article Scheduled]");
+
+		_updateElementInstancesJSON(
+			new Object[] {
+				HashMapBuilder.<String, Object>put(
+					"boost", 1
+				).put(
+					"fields", new String[] {"title_${context.language_id}^2"}
+				).put(
+					"fuzziness", "AUTO"
+				).put(
+					"keywords", "${keywords}"
+				).put(
+					"minimum_should_match", 0
+				).put(
+					"operator", "or"
+				).put(
+					"slop", 0
+				).put(
+					"type", "best_fields"
+				).build(),
+				null
+			},
+			new String[] {
+				"Text Match Over Multiple Fields", "Scheduling Aware"
+			});
+
+		_assertSearch("[]");
```

Comment:
Here and a few other places, we want to make sure that the filtering doesn't just remove results but also still allows some through. Add an unscheduled JournalArticle to this test that we would still expect to be returned in this last case.